README.md

```diff
@@ -34,6 +34,6 @@ The project is structured in a way to allow reuse of the code across various boa
 
 ## License
 
-This project is licensed under GPLv3.
+This project is licensed under GPLv3-only.
```

Comment:
If you are curious, this is opposed to `GPLv3-or-any-later-version` (if FSF releases a GPL 4, which probably won't happen)